src/components/editors/exportEditor.js

```diff
@@ -0,0 +1,84 @@
+import React, { useState } from 'react'
+import {
+  Dialog,
+  DialogTitle,
+  DialogContent,
+  DialogActions,
+  DialogContentText,
+  Typography,
+} from '@material-ui/core'
+
+import makeCssFile from '../../lib/generate'
+import { loadStorage, loadCss } from '../../lib/utils'
+
+import { Button } from '../SettingComponent'
+
+const ExportEditor = () => {
+  const [ settings, setSettings ] = useState( false )
+
+  const saveFile = () => {
+    const link = document.createElement( 'a' )
+    link.setAttribute( 'download', 'customTheme.css' )
+    link.href = makeCssFile()
+    document.body.appendChild( link )
```

Comment:
eeee any chance we can either render the react component for this OR just download the file? I'm sure you're allowed to download a file as long as the call has been triggered by a user event such as click